tests/logictest/suites/base/06_show/06_0001_show_create_table

```diff
@@ -13,25 +13,24 @@ SHOW CREATE TABLE `test`.`a`;
 ----
 a CREATE TABLE `a` (
   `a` BIGINT,
-  `b` INT DEFAULT 3,
-  `c` VARCHAR DEFAULT 'x',
+  `b` INT DEFAULT CAST(3 AS INT),
```

Comment:
Shall we ignore the explicit cast?

Reply:
Maybe the explicit cast is meaningful and will give user a hint (for the following case, there is precision truncation)

```sql
mysql> create table t(b int, a int default 0.1)
    -> ;
Query OK, 0 rows affected (0.03 sec)

mysql> show create table t;
+-------+--------------------------------------------------------------------------------+
| Table | Create Table                                                                   |
+-------+--------------------------------------------------------------------------------+
| t     | CREATE TABLE `t` (
  `b` INT,
  `a` INT DEFAULT CAST(0.1 AS INT)
) ENGINE=FUSE |
+-------+--------------------------------------------------------------------------------+
1 row in set (0.02 sec)
Read 0 rows, 0.00 B in 0.003 sec., 0 rows/sec., 0.00 B/sec.

mysql> insert into t values(3);
Query OK, 0 rows affected (0.04 sec)

mysql> select * from t;
+------+------+
| b    | a    |
+------+------+
|    3 |    0 |
+------+------+
1 row in set (0.04 sec)
Read 1 rows, 8.00 B in 0.007 sec., 135.24 rows/sec., 1.06 KiB/sec.

```